packages/server/src/adapters/aws-lambda/index.ts

```diff
@@ -52,10 +52,10 @@ function getHTTPMethod(event: APIGatewayEvent) {
 }
 function getPath(event: APIGatewayEvent) {
   if (isPayloadV1(event)) {
-    return event.path.slice(1);
+    return event.path.split('/')[event.path.split('/').length - 1];
```

Comment:
I’m not sure it’s safe to assume all users want to do this, I think it’s dependent on how api gateway is configured - which _might_ be in serverless framework, but it also might be serverless-components, or aws-cdk, or aws-sam, or cloudformation, or pulumi, or terraform, or point and click, or serverless-stack or…

Reply:
Also won’t this break for any users who use `/` in their procedure paths?

Reply:
> Can’t api gateway be configured with something like api/foo/bar/trpc/{proxy+}

That's exactly what this handles :) With the current code api/foo/bar/trpc/xxxx is passed to trpc as the query

>also capturing everything after the last slash wouldn’t work for my team’s router for example, which has procedures like abc/def/doSomething (usually via .merge('abc/', defRouter) or similar). This change would change the path to doSomething and would throw out the abc/def/ wouldn’t it?

This would do exactly that, thanks! You are right I can't see another way of doing this with path or rawPath without breaking your router. I suspect there is another API gateway event property I should be using here